conans/model/new_build_info.py

```diff
@@ -42,33 +41,34 @@ def __init__(self):
     @property
     def required_component_names(self):
         """ Names of the required components of the same package (not scoped with ::)"""
-        if not self.requires:
+        if self.requires is None:
```

Comment:
Did some minor review of this file while understanding it:

- In general, it is better and faster to check for ``None`` explicitly.